pkg/users/account-create-dialog.js

```diff
@@ -119,22 +119,22 @@ function validate_username(username, accounts) {
 function suggest_username(realname) {
     function remove_diacritics(str) {
         const translate_table = {
-            a :  '[àáâãäå]',
-            ae:  'æ',
-            c :  '[čç]',
-            d :  'ď',
-            e :  '[èéêë]',
-            i :  '[íìïî]',
-            l :  '[ĺľ]',
-            n :  '[ňñ]',
-            o :  '[òóôõö]',
-            oe:  'œ',
-            r :  '[ŕř]',
-            s :  'š',
-            t :  'ť',
-            u :  '[ùúůûűü]',
-            y :  '[ýÿ]',
-            z :  'ž',
+            a: '[àáâãäå]',
+            ae: 'æ',
+            c: '[čç]',
+            d: 'ď',
+            e: '[èéêë]',
+            i: '[íìïî]',
+            l: '[ĺľ]',
+            n: '[ňñ]',
+            o: '[òóôõö]',
+            oe: 'œ',
+            r: '[ŕř]',
+            s: 'š',
+            t: 'ť',
+            u: '[ùúůûűü]',
+            y: '[ýÿ]',
+            z: 'ž',
```

Comment:
These 16 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17876-20221104-114502-d7b5a827-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)